spec/system/candidate_interface/candidate_entering_degrees_spec.rb

```diff
@@ -259,7 +259,7 @@ def and_i_submit_the_add_another_degree_form
   end
 
   def then_i_can_check_my_additional_degree
-    expect(page).to have_content 'Masters Maths'
+    expect(page).to have_content 'Masters (Hons) Maths'
```

Comment:
'Masters (Hons)' can occur under the current implementation. Is that valid/correct?

Reply:
I think so 🤷